api/config/config.go

```diff
@@ -76,30 +78,24 @@ type UIConfig struct {
 	StaticPath string `validated:"required"`
 	IndexPath  string `validated:"required"`
 
-	FeastCoreAPI        string `json:"REACT_APP_FEAST_CORE_API"`
-	MerlinAPI           string `json:"REACT_APP_MERLIN_API"`
-	TuringAPI           string `json:"REACT_APP_TURING_API"`
 	ClockworkUIHomepage string `json:"REACT_APP_CLOCKWORK_UI_HOMEPAGE"`
-	FeastUIHomepage     string `json:"REACT_APP_FEAST_UI_HOMEPAGE"`
 	KubeflowUIHomepage  string `json:"REACT_APP_KUBEFLOW_UI_HOMEPAGE"`
```

Comment:
I don't suppose there are plans to refactor clockwork nor kubeflow to not consume env vars from MLP?

Reply:
I'll try to take this out in the next MR